apps/framework-cli/src/framework/python/templates.rs

```diff
@@ -71,19 +65,15 @@ pub static PYTHON_BASE_STREAMING_FUNCTION_TEMPLATE: &str = r#"
 # Add your models & start the development server to import these types
 {{source_import}}
 {{destination_import}}
-from dataclasses import dataclass
-from typing import Callable, Optional
+from moose_lib import StreamingFunction
+from typing import Optional
 from datetime import datetime
 
-@dataclass
-class Flow:
-    run: Callable
-
-def flow(activity: {{source}}) -> Optional[{{destination}}]:
+def fn(activity: {{source}}) -> Optional[{{destination}}]:
```

Comment:
@okane16 - do we want to the default name here to be `activity`?  I've re-named it `source` in the github analytics re-write.